docs/basic/attributes.md

```diff
@@ -206,6 +206,49 @@ fn main() {
 }
 ```
 
+Selain itu, kita juga bisa menggunakannya untuk conditional compilation based on profile.
```

Comment:
this is a great addition. I added my modification to ensure that we do not teach the user something that hasn't been explained in the previous chapter, for example, macro `cfg!()`. User might have confusion understanding it without additional details.